Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_PlayerPhysics.cs

```diff
@@ -0,0 +1,115 @@
+namespace VRTK
```

Comment:
to remove the unnecessary file you'll need to do a git remove `git rm <path-to-file>` then it will show up as a valid file commit (to remove) so when you do `git commit --amend` your commit will no longer have that file in it (and it will be like it never existed!)